### PR TITLE
HCS: Update topic

### PR DESCRIPTION
**Pull request by steven-sheehy**
- Implement update topic
- Add additional setters to all other consensus transactions
- Expand create and simple workflow examples

### CODE REVIEW

File: examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
         System.out.println("Created topic: " + topicId);
 
         var topicInfo = client.getTopicInfo(topicId);
         System.out.println("Retrieved topic info: " + topicInfo);
 
-        var transactionId = client.submitMessage(topicId, message.getBytes(Charset.forName("UTF-8")));
+        var transactionId = client.submitMessage(topicId, "Message #1".getBytes(Charset.forName("UTF-8")));
+        System.out.println("Submitted message: " + transactionId);
+
+        var newAccount = client.createAccount(newKey.getPublicKey(), 10);
```
Comment (steven-sheehy) [inline examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java:32]: Currently getting an error here `INSUFFICIENT_TX_FEE`. Need to resolve before merge

Comment (mike-burrage-hedera) [reply examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java:32]: client.setMaxTransactionFee(10_000_000L) before createAccount() should get past this.

File: src/main/java/com/hedera/hashgraph/sdk/Client.java [hunk 2]
```diff
@@ -40,15 +37,6 @@ public Client(Map<AccountId, String> nodes) {
...
-    public Client setMaxTransactionFee(@Nonnegative long maxTransactionFee) {
-        if (maxTransactionFee <= 0) {
-            throw new IllegalArgumentException("maxTransactionFee must be > 0");
-        }
-
-        this.maxTransactionFee = maxTransactionFee;
-        return this;
-    }
-
     public Client setOperator(AccountId operatorId, Ed25519PrivateKey operatorKey) {
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/hashgraph/sdk/Client.java]: Are these mainly changes from reformatting the code based on platform's intellij rules?

Comment (steven-sheehy) [reply src/main/java/com/hedera/hashgraph/sdk/Client.java]: Yes, I can undo if you like. It just auto-saves that way.

Comment (mike-burrage-hedera) [reply src/main/java/com/hedera/hashgraph/sdk/Client.java]: Nope, all good. Long as we can say that to LB on the main PR. If they want it undone, we'll do that then.

File: examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java (hunk 1 above)
Comment (steven-sheehy) [reply examples/src/main/java/com/hedera/hashgraph/sdk/examples/consensus/SimpleWorkflow.java:32]: This fixed it

**Pull request merged**